lightllm/server/router/manager.py

```diff
@@ -12,10 +13,12 @@
 from lightllm.utils.infer_utils import calculate_time
 from ..io_struct import BatchTokenIdOut, AbortReq
 
+_LOGGING_INTERVAL_SEC = 10
```

Comment:
10 sec period seems a little bit long to me.
How about make it configurable

Reply:
> 10 sec period seems a little bit long to me. How about make it configurable

done